lib/react_client.dart

```diff
@@ -355,20 +312,14 @@ ReactComponentFactory _registerComponent(ComponentFactory componentFactory, [Ite
     }, skipMethods)
   )]);
 
-  /**
-   * return ReactComponentFactory which produce react component with set props and children[s]
-   */
   return new ReactDartComponentFactoryProxy(reactComponentClass);
 }
 
-/**
- * A Dart function that creates React JS component instances for DOM components.
- */
+/// Creates ReactJS [Component] instances for DOM components.
```

Comment:
This creates ReactJS `ReactElement` instances, not Dart `[Component]` instances.